packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -37,9 +40,55 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
       activeAppearance,
       image: imageProps,
       badge: badgeProps,
+      initials: _initials,
     },
     state: {
       ...state,
     },
   };
 };
+
+/**
+ * A function which takes a name and returns initials
+ *
+ * @param name
+ * @returns initials. First letter of the word if only one word was provided.
+ * First letters of the first and last words if more words were provided.
+ * Words in braces, titles, special characters, parantheses and dashes should be ignored.
+ */
+export const getInitials = (name: string): string => {
+  if (!(name || validateAlphabeticalCharacters(name))) {
```

Comment:
>if (!(name || validateAlphabeticalCharacters(name))) [](http://example.com/codeflow?start=2&length=52)

super nit: I think we usually distribute the negation and get rid of the parentheses (apply DeMorgan's law)

so it would look like

`if (!name && !validateAlphabeticalCharacters(name))`

Reply:
Ok, I thought it was better without negation. But there are a lot of parentheses :) I'll fix it